docs/guides/setup/components/install_prereq.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
```

Comment:
Rather than duplicating this component, perhaps we can move the existing one out to a more common place so it can be shared. We may want to factor out `defaultPrerequisiteItems` so it is more general and reusable. Happy to pair if you'd like!

Reply:
Will be handled in the next PR as it requires more changes